examples/sessions/middleware.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const boltzmann = require('./boltzmann')
+
+module.exports = {
+  APP_MIDDLEWARE: [
+    [
+      boltzmann.middleware.applyCSRF,
+      {
+        cookieSecret: process.env.COOKIE_SECRET || "it's a secret to everyone".repeat(2),
+      },
+    ],
```

Comment:
I disagree with prettier about indentation and newlines here, but such is the fate of all code formatters